### DISCUSSION TITLE
Discourse no longer available - discussions moved here

### DISCUSSION THREAD

**Discussion by cklosters** [napframework/nap#22] [Announcements]
Unfortunately our discourse server went down and we're not in a position to bring it back up. Instead of creating a new instance we decided to move the general NAP discussion here. Apologies for the inconvenience! There is a backup and I will try to import previous topics if / when possible. Feel free to use this forum to start a conversation and ask questions! It won't go anywhere, I promise ;)

**Comment by jkukatzki** [comment]
Hello, I wanted to create a topic in Q&A but github tells me "You cant perform that action at this time." Maybe some permissions aren't set?
This might be how you do it: https://docs.github.com/en/organizations/managing-organization-settings/managing-discussion-creation-for-repositories-in-your-organization

Cheers.

**Comment by cklosters** [comment]
> Hello, I wanted to create a topic in Q&A but github tells me "You cant perform that action at this time." Maybe some permissions aren't set? This might be how you do it: https://docs.github.com/en/organizations/managing-organization-settings/managing-discussion-creation-for-repositories-in-your-organization
> 
> Cheers.

Thanks for pointing this out, I have given users with read access the right to create discussions.
